<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SequenceNumberOptionDialogWidget</class>
 <widget class="QWidget" name="SequenceNumberOptionDialogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>490</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string comment="Number of digits ">Digits:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="digits">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Custom Range</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string comment="Start index for custom range">Start:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="start">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string comment="stepping for custom range">Step:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="step">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="extensionAware">
     <property name="text">
      <string>Extension aware numbering</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="folderAware">
     <property name="text">
      <string>Folder aware numbering</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="counterAware">
     <property name="text">
      <string>File counter aware numbering</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="randomAware">
     <property name="text">
      <string>Random text aware numbering</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="counterAndExtAware">
     <property name="text">
      <string>File counter and extensions aware numbering</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="randomAndExtAware">
     <property name="text">
      <string>Random text and extensions aware numbering</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
